v4m-container/README.md

```diff
@@ -42,7 +42,30 @@ in a directory named `viya4-monitoring-kubernetes-<release_number>`.
 5. Change to the `viya4-monitoring-kubernetes` directory.
 6. Enter the command `git checkout <release_number>`. If you used the command `git clone --branch <my_branch> <https_url>` in Step 4 to specify the branch, release, or tag, you do not have to perform this step
 
-### Building the Docker Image
+### Populate v4m-container Directories
+
+#### Kubeconfig Directory
+
+To ensure that your kubeconfig files are available from within the Docker container, you can copy the kubeconfig files to the `./v4m-container/kubeconfig` directory or mount the file or directory during your `docker run` command.  By default, the `KUBECONFIG` environment variable is set to use a file called `config`; therefore, if multiple kubeconfig files are being provided, rename the one you would like to use to `config`.
+
+#### (Optional) user_dir Directory
+
+You can customize the Viya 4 Monitoring for Kubernetes deployment by editing files in a USER_DIR directory. See the [monitoring README](../monitoring/README.md) and [logging README](../logging/README.md) for detailed information about the customization process and about determining valid customization values.
+
+If you would like to customize your deployment, you will need to copy the contents of your USER_DIR directory to the `./v4m-container/user_dir` directory or mount the file or directory during your `docker run` command.
+
+### Mounting Files and Directories to your Docker Container
```

Comment:
Personal Preference: I feel like this section would fit more naturally with the "running the container" part instead of before the "building the container" section.